src/App.tsx

```diff
@@ -1,26 +1,59 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { IonApp, IonTabBar, IonTabButton, IonIcon, IonLabel, IonTabs, IonRouterOutlet } from "@ionic/react";
+import { Route, Redirect } from "react-router-dom";
+import { IonReactRouter } from "@ionic/react-router";
+import {
+  calendarOutline as calendarIcon,
+  wifiOutline as wifiIcon,
+  documentAttachOutline as docIcon,
+  peopleOutline as peopleIcon,
+  mapOutline as mapIcon,
+} from "ionicons/icons";
+import React from "react";
+import Agenda from "./pages/Agenda";
```

Comment:
Pierdoła ale zawsze. 

Fajnie jest mieć jakiś standard kolejności importów. Ja to zwykle robie tak że dziele je sobie na pare sekcji (na początku importu bibliotek/frameworku/wszystkiego z npma. 

Potem linijka przerwy i importy komponentów. Jak do tego dochodzi state albo jakieś pomocnicze pliku z funkcjami to daje je w kolejnej sekcji.

Jeżeli chodzi o kolejność to imo najlepiej sprawdza się alfabetycznie